tests/acceptance.suite.yml

```diff
@@ -27,7 +27,7 @@ modules:
         url: 'http://test.local'
         browser: chrome
         port: 4444
-        window_size: '1280x980'
+        window_size: '1366x980'
```

Comment:
Just curious. Why we need to to this?

Reply:
Aha I see. It makes sense. It looks like that after the addition the table became quite big. I don't know how many users use 1280px wide screens, but as I see from [the screenshot](https://d.pr/i/bH9mp7) it may at least look a bit broken for such users.
Maybe we could compress the information about subscribers counts into one column to save some space, but that is out of scope of this task. I guess @veljkho could check how the table works for users with smaller screens.